### PR TITLE
feat(Avatar): new component

### PR DESCRIPTION
**Pull request by GaroGabrielyan**
### Chromatic Build Summary 
**Build URL:** https://www.chromatic.com/build?appId=66daee7859a843dd87bfb3d7&number=74 
**Storybook URL:** https://66daee7859a843dd87bfb3d7-ximxkdkmhw.chromatic.com/

### CODE REVIEW

File: src/lib/atoms/Avatar/Avatar.tsx [hunk 1]
```diff
@@ -1,68 +1,45 @@
...
+        if (!fullName) {
+            setCutFirstAndLastName('');
+            return;
+        }
+        const [firstLetter, secondLetter] = fullName.split(' ');
+        setCutFirstAndLastName(`${firstLetter[0]} ${secondLetter ? secondLetter[0] : ''}`);
+    }, [fullName]);
+
     return (
         <div
```
Comment (GaroGabrielyan) [inline src/lib/atoms/Avatar/Avatar.tsx:32]: This is an interactive element and i think that enter must work on it as onClick

File: src/lib/atoms/Avatar/Avatar.stories.tsx [hunk 2]
```diff
@@ -1,46 +1,35 @@
...
-        children: args({ control: 'text', category: category.content }),
-        size: args({ control: 'select', category: category.appearance }),
-        shape: args({ control: 'select', defaultValue: 'circle', category: category.appearance }),
-        color: args({ control: 'select', defaultValue: 'default', category: category.appearance }),
-        icon: args({ control: 'text', category: category.content }),
-        src: args({ control: 'text', category: category.content }),
-        onClick: args({ control: false, category: category.action })
+        size: args({ control: 'select', ...propCategory.appearance }),
+        color: args({ control: 'select', ...propCategory.appearance }),
+        src: args({ control: 'text', ...propCategory.functionality }),
```
[the remark this answers is not in the document]
Comment (NarekArshakyan) [reply src/lib/atoms/Avatar/Avatar.stories.tsx:16]: why ?

**Review by AregSoft — APPROVED** [review]

File: src/lib/atoms/Avatar/Avatar.tsx [hunk 3]
```diff
@@ -1,68 +1,118 @@
...
+            parentClass={classNames(`avatar avatar_size_${size} avatar_color_${color}`, className, {
+                avatar_disabled: isDisabled
             })}
-            style={{ backgroundImage: src ? `url(${src})` : '' }}
             onClick={onClick}
         >
-            {!src && !icon && children && getInitials(children)}
-        </div>
+            {iconMock && !cutFirstAndLastName && iconMock}
+            {src && <img className="avatar__image" alt={'avatar'} src={src} />}
```
Comment (hamikhambardzumyan) [inline src/lib/atoms/Avatar/Avatar.tsx:112]: Here is exist issue in rendering `src` and `icon` will be rendered together handle logic

File: src/lib/atoms/Avatar/Avatar.test.tsx [hunk 4]
```diff
@@ -1,32 +1,79 @@
...
+describe('Avatar ', () => {
+    let setup: ReactWrapper<IAvatarProps>;
+    beforeEach(() => (setup = mount(<Avatar />)));
+    const mockFn = jest.fn();
 
-describe('Avatar Component', () => {
     it('renders without crashing', () => {
-        const wrapper = shallow(<Avatar />);
-        expect(wrapper.exists()).toBeTruthy();
+        console.log(setup.debug(), 88888);
```
Comment (hamikhambardzumyan) [inline src/lib/atoms/Avatar/Avatar.test.tsx:13]: Remove the log

File: src/lib/atoms/Avatar/Avatar.test.tsx [hunk 5]
```diff
@@ -1,32 +1,79 @@
...
+    it('renders isLoading prop correctly', () => {
+        const wrapper = setup.setProps({ isLoading: true });
+
+        expect(wrapper.find('skeleton')).toBeTruthy();
     });
 
-    it('renders with an icon when icon prop is provided', () => {
-        const wrapper = shallow(<Avatar icon="icon-test" />);
-        expect(wrapper.find('.user-avatar-c').hasClass('icon-test')).toBeTruthy();
+    it('renders onClick prop correctly', () => {
```
Comment (hamikhambardzumyan) [inline src/lib/atoms/Avatar/Avatar.test.tsx:48]: Follow the test writing guideline

**Review by hamikhambardzumyan — CHANGES_REQUESTED** [review]

File: src/lib/atoms/Avatar/Avatar.test.tsx [hunk 6]
```diff
@@ -1,32 +1,85 @@
-import React from 'react';
-import { shallow } from 'enzyme';
-import Avatar from './index';
+import React, { MouseEvent } from 'react';
+import { ReactWrapper, mount } from 'enzyme';
+import { Square } from '@geneui/icons';
```
Comment (hamikhambardzumyan) [inline src/lib/atoms/Avatar/Avatar.test.tsx:3]:
```suggestion
import { Square } from '@geneui/icons'; // TODO: replace with the person icon
```

**Review by hamikhambardzumyan — CHANGES_REQUESTED** [review]

**Review by hamikhambardzumyan — APPROVED** [review]

**Pull request merged**